Google.Api.Generator/Generation/SnippetCodeGenerator.cs

```diff
@@ -435,8 +435,8 @@ private IEnumerable<ObjectInitExpr> InitRequest()
                         makeRequest,
                         BlankLine,
                         "// Read streaming responses from server until complete",
-                        ResponseStream.WithInitializer(Response.Access(nameof(ServerStreamingBase<int>.ResponseStream))),
-                        While(Await(ResponseStream.Call(nameof(IAsyncEnumerator<int>.MoveNext))()))(
+                        ResponseStream.WithInitializer(Response.Call(nameof(ServerStreamingBase<int>.GetResponseStream))(Ctx.Type(typeof(CancellationToken)).Access(nameof(CancellationToken.None)))),
```

Comment:
I think there's a slight shortcut you can use: `Ctx.Type<CancellationToken>()`

Reply:
Ooh thanks, will do that.